iree/test/e2e/tosa_ops/if.mlir

```diff
@@ -0,0 +1,35 @@
+func @if_true_test() attributes { iree.module.export } {
+  %0 = iree.unfoldable_constant dense<true> : tensor<i1>
+  %1 = iree.unfoldable_constant dense<5> : tensor<i32>
+  %2 = "tosa.cond_if"(%0, %1) ( {
+  ^bb0(%arg2: tensor<i32>):  // no predecessors
```

Comment:
nit: where's `arg1`? :P (thoughout this file, only `arg2`?)

Reply:
Originally this was generated from a sample tosa file with arg0 and arg1 as inputs. Renamed to use arg0 as-per mlir default naming.